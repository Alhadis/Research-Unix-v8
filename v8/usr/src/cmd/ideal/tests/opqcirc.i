opqcirc
.nf
.IS
...libfile circle
main{
	conn -3 to 3;
	put circle {
		center = 0;
		radius = 0.5;
	};
	put circle {
		center = 0.5;
		radius = 1;
	};
	put circle {
		center = 3;
		radius = 1;
	};
	put circle {
		center = 0;
		radius = 1;
		opaque;
	};
}
.IE
