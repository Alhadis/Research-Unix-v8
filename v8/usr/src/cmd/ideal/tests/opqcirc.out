opqcirc
.nf
.IS
...maxx 4
...maxy 1
...minx -3
...miny -1
...circle 0 0 1
...arc 3 0 4 0 4 1.74846e-07 0 6.28319 1
...arc 0.5 0 1.5 0 0.25 0.968246 0 1.82348 1
...arc 0.5 0 0.25 -0.968246 1.5 1.74846e-07 4.45971 6.28319 1
...line 3 0 1 0
...line -1 0 -3 0
.IE
